### PR TITLE
script: upgrade uni lp geyser in order to sweep

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
solution for #759 and part of #747

### CODE REVIEW

File: scripts/issue/759/upgrade_uni_lp_geyser.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+def queue():
+    # queue the upgrade
+    DEV.badger.queue_timelock(
+        target_addr=DEV_PROXY_ADMIN.address,
+        signature='upgrade(address,address)',
+        data=encode_abi(['address', 'address'], [GEYSER.address, NEW_LOGIC]),
+        dump_dir=DUMP_DIR,
+        delay_in_days=4,
+    )
```
Comment (petrovska-petro) [inline scripts/issue/759/upgrade_uni_lp_geyser.py:37]: missing posting 

```suggestion
    )
    DEV.post_safe_tx(call_trace=True)
```

Comment (gosuto-inzasheru) [reply scripts/issue/759/upgrade_uni_lp_geyser.py:37]: thanks

**Review by petrovska-petro — APPROVED** [review]
upgrades logic to allow to sweeping out the badger. [compare files](https://www.diffchecker.com/o0iKd8vs). updates `recoverERC20`  and `getReward()` methods

- sim works sweeping out the badger amount ending up in trops ✅ 
- added other erc20 in the mix when exec the time lock to send to trops ✅

**Review by jalbrekt85 — APPROVED** [review]
new impl [addr](https://etherscan.io/address/0xf8C383A50984Ea50D9e343572F49a09152b3e674#code) lgtm. nice way of comparing storage vars before and after

**Review by sajanrajdev — APPROVED** [review]
Contract:
Logic diff can be found here: https://www.diffchecker.com/9C5JO6SK
- The `recoverERC20` is being modified to allow for the "reward token" (BADGER) to be swept - LGTM
- Removes a hotfix to the `getReward` function - No impact as the contract will be virtually deprecated after this operation

Scripts:
- Queuing LG and works as expected
- Executing will call `recoverERC20` atomically
- A simulation is included to confirm these actions
- Sweeps BADGER, AURA, USDC, WETG, WBTC from dev to Trops atomically
```
snapshot result for 0xB65cef03b9B89f99517643226d76e286ee999e77:
                  balance_before         balance_after              balance_delta
symbol
AURA    3,040.720934269983193963  0.000000000000000000  -3,040.720934269983193963
USDC      739.356929000000000000  0.000000000000000000    -739.356929000000000000
WETH        2.647848787301678387  0.000000000000000000      -2.647848787301678387
WBTC        0.014193600000000000  0.000000000000000000      -0.014193600000000000

snapshot result for 0x042B32Ac6b453485e357938bdC38e0340d4b9276:
                   balance_before              balance_after              balance_delta
symbol
BADGER  67,206.325067745439920154  84,633.084816216000124357  17,426.759748470560204203
AURA         0.000000000000000000   3,040.720934269983193963   3,040.720934269983193963
USDC         0.000000000000000000     739.356929000000000000     739.356929000000000000
WETH         0.473286343588024763       3.121135130889703150       2.647848787301678387
WBTC        31.239493270000000000      31.253686870000000000       0.014193600000000000
```

**Pull request merged**